src/platform/pantherlake/include/platform/drivers/alh.h

```diff
@@ -0,0 +1,19 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
```

Comment:
Please remove "All rights reserved", this is not the official guidance from Intel.

@lrgirdwo this needs to be fixed across all Intel contributions

Reply:
should be 2022-2024 anyways